web-elements/src/test/java/com/demoqa/ButtonsTest.java

```diff
@@ -0,0 +1,127 @@
+package com.demoqa;
+
+import com.demoqa.page.ButtonsPage;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+
+public class ButtonsTest extends BaseTest {
+    @Test
+    public void testPageIsOpenTest(){
+        ButtonsPage buttonsPage=new ButtonsPage(driver);
+        buttonsPage.open("buttons");
+        String currentURL = driver.getCurrentUrl();
+        String expectedURL = "https://demoqa.com/buttons";
+        assertEquals(currentURL,expectedURL);
+    }
+    @Test
+    public void doubleLeftClickByDoubleClickButtonTest(){
+        ButtonsPage buttonsPage=new ButtonsPage(driver);
+        buttonsPage.open("buttons");
+        buttonsPage.doubleClickLeft(buttonsPage.doubleClickButton);
```

Comment:
Let's discuss who is responsible for double click ) Is it page?
Can we use double-click on the other pages? If 'yes', please, create an additional class for some mouse function (something like MouseAction) and provide static methods which will be implemented this logic